docs/component/open-attestation.md

```diff
@@ -4,8 +4,311 @@ title: OpenAttestation Wrapper SDK (javascript)
 sidebar_label: Wrapper SDK (javascript)
 ---
 
-Overview
+The OpenAttestation SDK (javascript) is a npm module that allows you to manipulate OA documents programmatically. This is useful if you are building your own API or web components. Some common use cases where you need this module:
+- [Generating OA documents from external data sources programmatically](#wrapping-documents)
+- [Extracting data from OA documents programmatically](#retrieving-document-data)
+- [Programmatic selective disclosure of data fields from issued OA documents](#obfuscating-data)
 
-## Header
+This module does not provide the following functionality:
+- Programmatic verification of OA documents (refer to [Verification SDK (javascript)](/docs/component/oa-verify))
+- Encryption or decryption of OA documents (refer to [Encryption SDK (javascript)](/docs/component/oa-encryption))
+- Programmatic issuance/revocation of document on the Ethereum blockchain
 
-TBD
+## Installation
+
+1. Install [Node.js](https://nodejs.org/en/)
+1. Install the library: `npm install @govtechsg/open-attestation`
+
+## Usage
+
+### Wrapping documents
```

Comment:
Some details on what we mean by 'wrapping' will be useful i.e. What is wrapping?

Reply:
I will add a note, it depends if we explain it somewhere else